src/Helper/Helper_Abstract_Options.php

```diff
@@ -1545,10 +1546,13 @@ public function multicheck_callback( $args ) {
 				$args['multi-option'] = $key;
 
 				$enabled = $this->get_form_value( $args );
+				$label = esc_html( $option ) ;
+				$aria_checked = !empty( $enabled ) ? 'aria-checked="true"' : 'aria-checked="false"';
+				$aria_label = str_replace( '_', ' ',$label );
 
 				echo '<div class="gfpdf-settings-multicheck-wrapper">';
-				echo '<input name="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" id="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" class="gfpdf_settings_' . $args['id'] . ' ' . $class . '" type="checkbox" value="' . $args['multi-key'] . '" ' . checked( $key, $enabled, false ) . ' ' . $required . ' />&nbsp;';
-				echo '<label for="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']">' . esc_html( $option ) . '</label>';
+				echo '<input name="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" id="gfpdf_settings[' . $args['id'] . '][' . $args['multi-key'] . ']" class="gfpdf_settings_' . $args['id'] . ' ' . $class . '" type="checkbox" value="' . $args['multi-key'] . '" ' . checked( $key, $enabled, false ) . ' ' . $required . ' role="checkbox" ' . $aria_checked . ' />&nbsp;';
```

Comment:
This is an invalid use of `aria-checked`. See:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role